src/test/java/edu/cornell/kfs/sys/batch/service/impl/CuBatchInputFileServiceImplTest.java

```diff
@@ -166,7 +166,6 @@ private final void checkForDoneFile(File batchFile, boolean isExist) {
     private class MockDateTimeService extends DateTimeServiceImpl {
```

Comment:
Since you're removing the `TestDateTimeServiceImpl` class, it would be good to remove this nested `MockDateTimeService` class as well.

Reply:
Good point

Reply:
Updated